ChangeLog.md

```diff
@@ -14,6 +14,7 @@
 
 - Add default value consistently for parameters #1164
 - Make `content_type` param keyword-only if there are multiple content types #1167
+- Add flag `--default-optional-constants-to-none` for optional constant parameters #1171
```

Comment:
Should have picked this up in the first review, but we should do a minor version bump. We should add two entries, one under "New Features", and one other "Breaking Changes in Version Tolerant Generation".

In "New Features", we add this entry, saying "default optional constant parameters to None with this flag", and in "Brekaing Changes in Version Tolerant Generation" we say that by default we generate with optional constant parameters as None now by defaulting this flag to true